Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponentController.cpp

```diff
@@ -47,6 +47,9 @@ namespace MiniAudio
     {
         m_entityComponentIdPair = entityComponentIdPair;
 
+        m_config.m_innerAngleInRadians = m_config.m_innerAngleInDegrees * AZ::Constants::TwoPi / 360.f;
```

Comment:
(NIT) There is a `constexpr` `AZ::RadToDeg` and `AZ::DegToRad` in AzCore/Math/MathUtils.h fwiw

Reply:
Thanks for the suggestion, incorporated this in 4000e5f7185e1f1267d9cf2d606213b624e2aea2.